swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
To facilitate integration and testing. It's great documentation format too. We wouldn't need to expose it in production if we don't want to.